transferui/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
     implementation project(":transfermethodrepository")
     implementation project(":userrepository")
     implementation project(":commonui")
+    implementation project(":userbalancerepository")
```

Comment:
Why are we creating a new repository module? It should be part of `userrepository` module
Similar to how `receiptrepository` module has repository for `PrepaidCard` and `User`
Create `UserBalanceRepository` inside `com.hyperwallet.android.ui.user.repository`

I will review rest of the PRs after this is fixed